Products/CMFPlone/controlpanel/browser/redirects.py

```diff
@@ -190,20 +190,34 @@ def __init__(self, query="", created="", manual=""):
             else:
                 manual = ""
         if created:
+            end = created
```

Comment:
This should be 
```suggestion
            start = created
```
to match the previous implementation.

Reply:
No, sorry, I had it wrong.